addons/base_vat/models/res_partner.py

```diff
@@ -382,6 +384,8 @@ def check_vat_nl(self, vat):
             return True
 
         vat = clean(vat, ' -.').upper().strip()
+        if not vat.startswith('NL'):
+            vat = 'NL' + vat
```

Comment:
This is really weird. Aren't we supposed to check with the partner's country code if it does not work without it anyway? We should not need these two lines here.

Reply:
Indeed, you can argue that the country code is unseparable from the VAT code. I can refactor and make sure it works for NL but I'm not sure about the other localizations of course.

Reply:
Feel free to make a proposal; but don't forget we are in a stable version here ;) It's just that this condition to add NL to the code is clearly just a workaround for a bug, and we can't merge that, we need to to keep more control on what we do.